rastervision/backend/model_defaults.json

```diff
@@ -97,6 +97,7 @@
                 "trainBatchSize": 24,
                 "trainCropSize": [257, 257],
                 "trainingNumberOfSteps": 35410,
+		"numClones": 1,
```

Comment:
`numClones` is not indented the same way as the other fields (here and the instance below).

Reply:
Needed to be untabified.